microsoft-edge/extensions-chromium/developer-guide/sidebar.md

```diff
@@ -38,7 +36,7 @@ Extensions can optionally use the sidebar API to show a custom UI in the Microso
 
 Features of the Sidebar API include:
 
-* The sidebar remains open while navigating between tabs.
+* The sidebar remains open while navigating between tabs.  Known issue: The sidebar is not automatically displayed again when the user switches to a tab in which the sidebar was previously open ([Issue #142](https://github.com/microsoft/MicrosoftEdge-Extensions/issues/142)).<!-- todo: update -->
```

Comment:
is there a faq to also link to?
2x
@bhuvanapriyap

Reply:
no @mikehoffms , there is no faq as of now